Meta/run.sh

```diff
@@ -126,6 +126,22 @@ elif [ "$SERENITY_RUN" = "qcmd" ]; then
         -device e1000,netdev=breh \
         -kernel Kernel/Kernel \
         -append "${SERENITY_KERNEL_CMDLINE}"
+elif [ "$SERENITY_RUN" = "ci" ]; then
+    # Meta/run.sh ci: qemu in text mode
+    echo "Running QEMU in CI"
+    "$SERENITY_QEMU_BIN" \
+        $SERENITY_EXTRA_QEMU_ARGS \
+        -s -m $SERENITY_RAM_SIZE \
+        -cpu $SERENITY_QEMU_CPU \
+        -d guest_errors \
+        -smp 2 \
+        -drive file=${SERENITY_DISK_IMAGE},format=raw,index=0,media=disk \
+        -device ich9-ahci \
+        -nographic \
+        -display none \
+        -debugcon file:debug.log \
```

Comment:
Should be the same place as `e1000.pcap`, in `Build/`

Reply:
Re your intro comment: It's hanging in the actions because there legitimately is no output because the shell is hung in the kernel -- see #5541.  In a lucky run, the thing does complete all the way. And then dump the debug.log in the next step.

The way the qemu options work is whatever we write to /dev/ttyS0 will be sent to stdout, and the kernel log is sent to debug.log. So we see the test resuls scroll by, then then debug.log is cat-ed in the next !cancelled() step.

Reply:
Ah and yes, it will drop debug.log in $PWD. Which will likely be Build/ so no gitignore needed imo.